test/performance/serializing_test.rb

```diff
@@ -0,0 +1,11 @@
+require "benchmark_helper"
+
+class SerializingTest < ActionDispatch::PerformanceTest
+  # Refer to the documentation for all available options
+  # self.profile_options = { runs: 5, metrics: [:wall_time, :memory],
+  #                          output: 'tmp/performance', formats: [:flat] }
+
+  test "checkins_page" do
+    get "/users/3/checkins", headers: { "X-Api-Key" => Developer.first.api_key }
```

Comment:
I'm trying to write my own performance tests so I tried using this as an example. However this performance test appears to 404 (after redirecting)?